x-pack/plugins/cloud_security_posture/server/plugin.test.ts

```diff
@@ -222,65 +213,6 @@ describe('Cloud Security Posture Plugin', () => {
       expect(spy).toHaveBeenCalledTimes(0);
     });
 
-    it('packagePolicyPostCreate should return the updated packagePolicy', async () => {
-      fleetMock.packageService.asInternalUser.getInstallation.mockImplementationOnce(
-        async (): Promise<Installation | undefined> => {
-          return;
-        }
-      );
-
-      fleetMock.packagePolicyService.update.mockImplementation(
-        (
-          soClient: SavedObjectsClientContract,
-          esClient: ElasticsearchClient,
-          id: string,
-          packagePolicyUpdate: UpdatePackagePolicy
-        ): Promise<PackagePolicy> => {
-          // @ts-expect-error 2322
-          return packagePolicyUpdate;
-        }
-      );
-
-      const packageMock = createPackagePolicyMock();
-      packageMock.package!.name = CLOUD_SECURITY_POSTURE_PACKAGE_NAME;
-      packageMock.vars = { runtimeCfg: { type: 'foo' } };
-
-      const packagePolicyPostCreateCallbacks: PostPackagePolicyPostCreateCallback[] = [];
-      fleetMock.registerExternalCallback.mockImplementation((...args) => {
-        if (args[0] === 'packagePolicyPostCreate') {
-          packagePolicyPostCreateCallbacks.push(args[1]);
-        }
-      });
-
-      const context = coreMock.createPluginInitializerContext<unknown>();
-      plugin = new CspPlugin(context);
-      const spy = jest.spyOn(plugin, 'initialize').mockImplementation();
-
-      // Act
-      await plugin.start(coreMock.createStart(), mockPlugins);
-      await mockPlugins.fleet.fleetSetupCompleted();
-
-      // Assert
-      expect(fleetMock.packageService.asInternalUser.getInstallation).toHaveBeenCalledTimes(1);
-      expect(spy).toHaveBeenCalledTimes(0);
-
-      expect(packagePolicyPostCreateCallbacks.length).toBeGreaterThan(0);
-
-      for (const cb of packagePolicyPostCreateCallbacks) {
-        const updatedPackagePolicy = await cb(
-          packageMock,
-          contextMock,
-          httpServerMock.createKibanaRequest()
-        );
-        if (fleetMock.packagePolicyService.update.mock.calls.length) {
-          expect(updatedPackagePolicy).toHaveProperty('vars');
-          expect(updatedPackagePolicy.vars).toHaveProperty('runtimeCfg');
-          expect(updatedPackagePolicy.vars!.runtimeCfg).toHaveProperty('value');
-        }
-      }
-      expect(fleetMock.packagePolicyService.update).toHaveBeenCalledTimes(1);
-    });
```

Comment:
the test is still valid, please revert

we should remove only the assertions for setting `runtimeCfg` (lines 269-282)

Reply:
I changed the test so it will be valid 💪